client_firmware/client_firmware.ino

```diff
@@ -1,17 +1,18 @@
 #include <EEPROM.h>
 #include <EtherCard.h>
-#include <IPAddress.h>
 #include <avr/wdt.h>
 
 #define CSPIN 10
 // NOTE: All of these timers are in Milliseconds!
-// Ping our target every 2 seconds
-#define PINGER_INTERVAL 2000
+// Ping our target every 60 seconds
+#define PINGER_INTERVAL 60000
```

Comment:
Ah. Broadcasts are already picked up; the NIC has to pick them up to respond to ARP requests. There's a bit that needs to be flipped in the HW to enable multicast reception though.